app/src/main/java/net/sourceforge/opencamera/MainActivity.java

```diff
@@ -729,6 +729,9 @@ public void onInit(int status) {
             notificationManager.createNotificationChannel(channel);
         }
 
+        // start RecSync leader-client setup
+        applicationInterface.startSoftwareSync();
```

Comment:
So startSoftwareSync is called even if the recsync checkbox is switched off? Or I missed some other check

Reply:
Oh, ok, I see, this seems to be not implemented yet (later in code)